### ISSUE TITLE
Not working with Unity 2022 VRCSDK Beta

### ISSUE THREAD

**Issue by Donad678** [d4rkc0d3r/d4rkAvatarOptimizer#65]
The optimizer fails on Unity 2022 with the VRCSDK 3.5.0 with following error

```
InvalidOperationException: Avatar is null.
UnityEngine.Animator.GetBoneTransform (UnityEngine.HumanBodyBones humanBoneId) (at <9802f153e66043958995cd88a8668a6e>:0)
d4rkAvatarOptimizer.FindAllMovingTransforms () (at ./Packages/d4rkpl4y3r.d4rkavataroptimizer/Editor/d4rkAvatarOptimizer.cs:2291)
d4rkAvatarOptimizer.FindAllUnmovingTransforms () (at ./Packages/d4rkpl4y3r.d4rkavataroptimizer/Editor/d4rkAvatarOptimizer.cs:2346)
d4rkAvatarOptimizer.FindMovingParent () (at ./Packages/d4rkpl4y3r.d4rkavataroptimizer/Editor/d4rkAvatarOptimizer.cs:3425)
d4rkAvatarOptimizer.CombineSkinnedMeshes () (at ./Packages/d4rkpl4y3r.d4rkavataroptimizer/Editor/d4rkAvatarOptimizer.cs:3444)
d4rkAvatarOptimizer.Optimize () (at ./Packages/d4rkpl4y3r.d4rkavataroptimizer/Editor/d4rkAvatarOptimizer.cs:149)
d4rkAvatarOptimizerEditor.OnInspectorGUI () (at ./Packages/d4rkpl4y3r.d4rkavataroptimizer/Editor/d4rkAvatarOptimizerEditor.cs:137)
UnityEditor.UIElements.InspectorElement+<>c__DisplayClass72_0.<CreateInspectorElementUsingIMGUI>b__0 () (at <347e3e2bef8c4deb82c9790c6e198135>:0)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr, Boolean&)
```

**Comment by d4rkc0d3r** [comment]
You have some extra animator component somewhere on your avatar. If you get rid of that, it should work.  
I already have a fix for this but have currently also a half finished feature in the works so it'll be a bit until the fix ships.

**Comment by Donad678** [comment]
Thank you, that fixed it.

**Comment by d4rkc0d3r** [comment]
fixed in 3.3.0
